src/iosrv/srv.c

```diff
@@ -800,7 +814,7 @@ dss_xstreams_init()
 	/* start system service XS */
 	for (i = 0; i < dss_sys_xs_nr; i++) {
 		xs_id = i;
-		rc = dss_start_xs_id(xs_id);
+		rc = dss_start_one_xstream(sys_xs_obj->allowed_cpuset, xs_id);
```

Comment:
here you create two sys XS and bind to the same core, but the following dss_start_xs_id() below (L811) it will leave one core unused as its xi_id bumped up two.
For example assume dss_core_offset is 0, then your two sys XS will use core 0, and the first IO main XS will use core 2 and leave core 1 unused, this should be fixed.